config/boards/station-m2.wip

```diff
@@ -0,0 +1,13 @@
+# Rockchip RK3566
+BOARD_NAME="Station M2"
+BOARDFAMILY="rk35xx"
+BOOTCONFIG="firefly-m2-rk3568_defconfig"
+BOOT_SOC_NAME="rk3568"
```

Comment:
Please rename it to BOOT_SOC, let's not introduce additional variables.
Especially that this new variable will not work with the BOOT_SOC based conditionals.

I added a suggestion in rockchip64_common.inc how to make this variable overridable.

Reply:
I will change this after checking how the override will work.